src/common/requests.ts

```diff
@@ -27,205 +27,204 @@ export function makeRequests(
 
 function exportRequestsType() {
   return ts.factory.createTypeAliasDeclaration(
-    undefined,
-    [ts.factory.createToken(ts.SyntaxKind.ExportKeyword)],
-    ts.factory.createIdentifier("Requests"),
-    undefined,
-    ts.factory.createTypeReferenceNode(
-      ts.factory.createIdentifier("ReturnType"),
-      [
-        ts.factory.createTypeQueryNode(
-          ts.factory.createIdentifier("makeRequests"),
-          undefined
-        ),
-      ]
-    )
+      undefined,
+      [ts.factory.createToken(ts.SyntaxKind.ExportKeyword)],
+      ts.factory.createIdentifier("Requests"),
+      undefined,
+      ts.factory.createTypeReferenceNode(
+          ts.factory.createIdentifier("ReturnType"),
```

Comment:
There's a lot of whitespace changes which are making the PR much bigger than it should be. Can you revert all the whitespace changes please?

Reply:
okay, sorry about that, my formatter